db/migrate/20161209204400_create_versions.rb

```diff
@@ -1,12 +1,14 @@
 class CreateVersions < ActiveRecord::Migration
   def change
     create_table :versions do |t|
-      t.belongs_to :user, index: true, null: false
-      
-      t.integer :version, index: true, null: false
-      t.boolean :production, index: true, default: false, null: false
```

Comment:
@mphprogrammer production used to have an index but doesn't anymore. is this intentional?